hammr/commands/migration/migration.py

```diff
@@ -48,8 +60,170 @@ def do_list(self, args):
             printer.out("ERROR: In Arguments: " + str(e), printer.ERROR)
             self.help_list()
         except Exception as e:
-            return handle_uforge_exception(e)
+            return hammr_utils.handle_uforge_exception(e)
 
     def help_list(self):
         doParser = self.arg_list()
         doParser.print_help()
+
+    def arg_launch(self):
+        doParser = ArgumentParser(prog=self.cmd_name + " launch",
+                                  add_help=True,
+                                  description="Launches a migration")
+        mandatory = doParser.add_argument_group("mandatory arguments")
+        mandatory.add_argument('--file',
+                               dest='file',
+                               required=True,
+                               help="yaml/json file providing the migration parameters")
+        return doParser
+
+    def do_launch(self, args):
+        try:
+            # add arguments
+            doParser = self.arg_launch()
+
+            try:
+                doArgs = doParser.parse_args(shlex.split(args))
+            except SystemExit:
+                return 2
+
+            migrationConfig = migration_utils.check_migration_configuration(doArgs.file)
+            targetFormat, targetPlatform = migration_utils.check_target(self.api, self.login, migrationConfig)
+            myPublishImage = migration_utils.check_publish_image(migrationConfig, targetFormat, doArgs.file)
+            myCredAccount = migration_utils.check_credAccount(self.api, self.login, migrationConfig, targetPlatform)
+
+            printer.out("file: " + doArgs.file + " is validated", "OK")
+
+            myMigration = self.createMigration(migrationConfig, targetFormat, myCredAccount, myPublishImage)
+
+            # call UForge API
+            self.api.Users(self.login).Migrations.Create(body=myMigration, element_name="ns1:migration")
+
+            # download migration binary
+            local_uforge_migration_path = self.download_migration_binary()
```

Comment:
style are mixed for variable names: myVar and my_var

Reply:
ok I switch to Camel case